packages/gafl-webapp-service/src/pages/summary/contact-summary/__tests__/route.spec.js

```diff
@@ -158,23 +156,23 @@ describe('contact-summary > route', () => {
   })
 
   it.each([
-    { preferredMethodOfNewsletter: HOW_CONTACTED.email, expectedReturn: mssgs.yes, action: 'sent' },
-    { preferredMethodOfNewsletter: HOW_CONTACTED.text, expectedReturn: mssgs.yes, action: 'sent' },
-    { preferredMethodOfNewsletter: HOW_CONTACTED.post, expectedReturn: mssgs.yes, action: 'sent' }
-  ])(
-    'newsletter is $action when preferredMethodOfNewletter is $preferredMethodOfNewsletter',
-    async ({ preferredMethodOfNewsletter, expectedReturn }) => {
-      const samplePermission = getMockPermission({
-        preferredMethodOfNewsletter
+    ['aye', HOW_CONTACTED.email, 'yes'],
+    ['negative, Ghost Rider', HOW_CONTACTED.none, 'no']
```

Comment:
Please will you change the 'aye' and 'negative, Ghost Rider' to be something that's not in the mock catalog already. Then we're done!